sr_multi_description/urdf/left_srhand_lite_ur5e.urdf.xacro

```diff
@@ -10,7 +10,7 @@
     <joint name="world_joint" type="fixed">
         <parent link="world"/>
         <child link="la_base_link"/>
-        <origin xyz="0.0 0.0 $(arg initial_z)" rpy="0.0 0.0 0.0"/>
+        <origin xyz="0.0 0.0 $(arg initial_z)" rpy="0.0 0.0 {pi}"/>
```

Comment:
In some other files I see `${pi}`. Is syntax without dollar also correct?

Reply:
no its wrong but its fixed on next commit